src/cmd/dev.go

```diff
@@ -0,0 +1,119 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2023-Present The UDS Authors
+
+// Package cmd contains the CLI commands for UDS.
+package cmd
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+
+	tea "github.com/charmbracelet/bubbletea"
+
+	"github.com/defenseunicorns/uds-cli/src/config"
+	"github.com/defenseunicorns/uds-cli/src/config/lang"
+	"github.com/defenseunicorns/uds-cli/src/pkg/bundle"
+	"github.com/defenseunicorns/uds-cli/src/pkg/bundle/tui/deploy"
+	"github.com/defenseunicorns/zarf/src/pkg/message"
+	"github.com/defenseunicorns/zarf/src/pkg/utils/helpers"
+	"github.com/spf13/cobra"
+	"golang.org/x/term"
+)
+
+var devCmd = &cobra.Command{
+	Use:   "dev",
+	Short: lang.CmdDevShort,
+}
+
+var devDeployCmd = &cobra.Command{
+	Use:   "deploy",
+	Args:  cobra.MaximumNArgs(1),
+	Short: lang.CmdDevDeployShort,
+	PreRun: func(_ *cobra.Command, args []string) {
+		pathToBundleFile := ""
+		if len(args) > 0 {
+			if !helpers.IsDir(args[0]) {
+				message.Fatalf(nil, "(%q) is not a valid path to a directory", args[0])
+			}
+			pathToBundleFile = filepath.Join(args[0])
+		}
+		// Handle .yaml or .yml
+		bundleYml := strings.Replace(config.BundleYAML, ".yaml", ".yml", 1)
```

Comment:
Since we're now handling `yaml` vs `yml` in multiple locations, can we break this out into a function?